internal/core/src/query/SearchBruteForce.cpp

```diff
@@ -50,6 +52,13 @@ BruteForceSearch(const dataset::SearchDataset& dataset,
         auto dim = dataset.dim;
         auto topk = dataset.topk;
 
+        bool isFp16 = false;
+        if (data_type == DataType::VECTOR_FLOAT16)
+        {
+            std::cout << "BruteForceSearch: data_type is VECTOR_FLOAT16" << std::endl;
+            isFp16 = true;
+        }
+
```

Comment:
What is this code trying to do?

Reply:
It's an argument to a function in knowhere, but I haven't finished the knowhere part yet. So it's useless right now,